packages/client/components/TaskFooterIntegrateMenu.tsx

```diff
@@ -23,38 +22,83 @@ const makePlaceholder = (hasGitHub: boolean, hasAtlassian: boolean) => {
   return `Search ${names.join(' & ')}`
 }
 
+type Integrations = NonNullable<TaskFooterIntegrateMenu_viewer['membership']>['integrations']
+
+const isIntegrated = (integrations: Integrations) => {
+  const {atlassian, github} = integrations
+  const hasAtlassian = atlassian?.isActive ?? false
+  const hasGitHub = github?.isActive ?? false
+  return hasAtlassian || hasGitHub
+    ? {
+        hasAtlassian,
+        hasGitHub
+      }
+    : null
+}
+
 const TaskFooterIntegrateMenu = (props: Props) => {
   const {menuProps, mutationProps, task, viewer} = props
-  const {id: viewerId, teamMember} = viewer
+  const {id: viewerId, membership, teamMember} = viewer
```

Comment:
nit-- let's rename teamMember to assigneeTeamMember or something similar & membership to viewerTeamMember